libmscore/chordrest.cpp

```diff
@@ -556,6 +550,24 @@ Element* ChordRest::drop(EditData& data)
                   score()->undoAddElement(e);
                   return e;
                   }
+            case ElementType::INSTRUMENT_CHANGE:
+                  if (e->isInstrumentChange() && part()->instruments()->find(tick().ticks()) != part()->instruments()->end()) {
```

Comment:
Thank you, I'll remove the isInstrumentChange(). I've tested it, and it doesn't block if you add to the first bar of the score.

Reply:
For clarity, this was an existing piece of code, that had a fall-through in it, hence the check. I rearranged it so that the above items didn't fall-through to it, so I could add some more specific code for instrument changes.

Reply:
Sorry, when you say it 'doesn't block', you mean that block doesn't run, right?